docs/source/conf.py

```diff
@@ -112,7 +112,7 @@
 
 # Theme options
 html_theme_options = {
-    "header_links_before_dropdown": 7,
+    "header_links_before_dropdown": 8,
```

Comment:
Looks poorly on middle-resolution screens. I would reduce the number of header links